python/tvm/script/__init__.py

```diff
@@ -14,8 +14,5 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-"""TVM Script APIs of TVM Python Package, aimed to support TIR"""
-
-from . import tir
-
-from .parser import ir_module, from_source
+"""TVM Script APIs of TVM Python Package"""
+from .parser import ir, ir_module, parse as from_source, tir
```

Comment:
I'm curious about this import. So `ir` becomes `from_source` and `ir_module` becomes `tir`? It doesn't make a lot sense to me, especially because we import three names as two names.

Reply:
separate by commas, `parse` becomes `from_source` while others are not renamed

Reply:
Oh I see. Thanks for the explanation :-)